### ISSUE TITLE
Release 5.1.0 cannot load 'SQLite.Interop.dll'

### ISSUE THREAD

**Issue by RedX2501** [jakubgarfield/Bonobo-Git-Server#393]
I just downloaded and installed the latest version. When trying to load the site I get the unhelpful page:
```
Runtime Error 
Description: An exception occurred while processing your request. Additionally, another exception occurred while executing the custom error page for the first exception. The request has been terminated. 
```

After disabling custom error pages I get:

```
Unable to load DLL 'SQLite.Interop.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E) 
Description: An unhandled exception occurred during the execution of the current web request. Please review the stack trace for more information about the error and where it originated in the code. 

Exception Details: System.DllNotFoundException: Unable to load DLL 'SQLite.Interop.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)
```

I could not find this DLL anywhere, not even on the older 3.4.3 release I have. Although System.Data.SQLite.dll, System.Data.SQLite.EF6.dll and System.Data.SQLite.Linq.dll are present in bonobo's bin dir.

I had only version 3.4.3 running previous to this upgrade.

**Comment by RedX2501** [comment]
some more insight.

The 5.0.1 version I downloaded last week for testing purposes )but was not working due to the root path redirect bug= hat the dlls under bin/bin/{x86,x64}. I copied it to that folder in the new version 5.1.0 but it did not work.

The older 3.4.3 installation had it under Bonobo.Git.server directly. Copying it to that localtion in 5.1.0 does make it work but I guess that is not the intended location.
